transactions/scripts/get_nft_metadata.cdc

```diff
@@ -0,0 +1,49 @@
+import TopShot from 0xTOPSHOTADDRESS
+import MetadataViews from 0xMETADATAVIEWSADDRESS
+
+pub struct NFT {
+    pub let name: String
+    pub let description: String
+    pub let thumbnail: String
+    pub let owner: Address
+    pub let type: String
+
+    init(
+        name: String,
+        description: String,
+        thumbnail: String,
+        owner: Address,
+        nftType: String,
+    ) {
+        self.name = name
+        self.description = description
+        self.thumbnail = thumbnail
+        self.owner = owner
+        self.type = nftType
+    }
+}
+
+pub fun main(address: Address, id: UInt64): NFT {
+    let account = getAccount(address)
+
+    let collectionRef = account.getCapability(/public/MomentCollection)
+                            .borrow<&{TopShot.MomentCollectionPublic}>()!
+
+    let nft = collectionRef.borrowMoment(id: id)!
+    
+    // Get the basic display information for this NFT
+    let view = nft.resolveView(Type<MetadataViews.Display>())!
```

Comment:
i thought this script will be returning the `TopShotMomentsMetadataView` view.

Reply:
Good call. Made a new script for TopShotMomentMetadataView view. Thanks.